src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import { Link } from 'react-router-dom'
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
```

Comment:
You can get rid of the outer `div` and change this one to: 
```suggestion
      <nav className="App">
```